src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -1164,6 +1172,54 @@ private void ProcessDownloadQueueLocked()
             this.logger.LogTrace("(-)");
         }
 
+        /// <summary>
+        /// Returns <c>true</c> if consensus' height is within <see cref="ConsensusIsConsideredToBeSyncedMargin"/>
+        /// blocks from the best tip's height.
+        /// </summary>
+        /// <remarks>Should be locked by <see cref="peerLock"/></remarks>
+        private bool IsConsensusConsideredToBeSyncedLocked()
```

Comment:
The `remarks` states it should be locked but it is not?

Reply:
It's always called under the peerLock

Reply:
Ah of course ;) Cool 👍